src/frontend/csrs-portal/src/app/auth/auth-config.module.ts

```diff
@@ -1,34 +1,48 @@
+/* eslint-disable arrow-body-style */
+import { HttpClient } from '@angular/common/http';
 import { NgModule } from '@angular/core';
-import { AuthModule, LogLevel } from 'angular-auth-oidc-client';
+import { AuthModule, LogLevel, StsConfigHttpLoader, StsConfigLoader } from 'angular-auth-oidc-client';
+import { map } from 'rxjs/operators';
 
+export const httpLoaderFactory = (httpClient: HttpClient) => {
+  const config$ = httpClient.get<any>(`assets/oidc.json`).pipe(
+    map((customConfig: any) => {
+      return {
+        triggerAuthorizationResultEvent: true,
+        postLoginRoute: customConfig.postLoginRoute,
+        logLevel: LogLevel.Debug,
+        historyCleanupOff: true,
```

Comment:
Hey ... may be turn this off for now until we find that it is an absolute must to track those for sort of issue resolution (normal or security related).

Reply:
It was commented and left for later refactoring.

Reply:
line historyCleanupOff: true was deleted. it was not necessary for portal application.